gtsam/linear/tests/testJacobianFactor.cpp

```diff
@@ -55,7 +54,7 @@ TEST(JacobianFactor, constructors_and_accessors)
   {
     // b vector only constructor
     JacobianFactor expected(
-      boost::make_iterator_range(terms.begin(), terms.begin()), b);
+      std::vector(terms.begin(), terms.begin()), b);
```

Comment:
is there not a constructor that takes iterators anyway? I thought there definitely was one in Factor.h, but maybe not exposed.